core/common.go

```diff
@@ -292,3 +296,31 @@ func buildAuthConfiguration(registry string) docker.AuthConfiguration {
 
 	return auth
 }
+
+const HashmeTagName = "hash"
+
+func getHash(t reflect.Type, v reflect.Value, hash *string) {
```

Comment:
That library does not support choosing the hashed elements but rather has an ignore list. I could switch but it would require ignoring instead of specifically adding the fields. I have no problem doing this if you prefer